.github/workflows/sync-branch.yml

```diff
@@ -0,0 +1,27 @@
+name: Sync branches
+
+on:
+  schedule:
+    - cron: "0 */2 * * 1-5" # Cron timezone is in UTC
+
+env:
+  TARGET_BRANCH: test-next
+  SOURCE_BRANCH: main
+
+jobs:
+  sync-branches:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'microsoft'
+    steps: 
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      
+      - name: Install dependencies
+        run: npm install @fluidframework/build-tools
```

Comment:
See https://github.com/microsoft/FluidFramework/actions/runs/5468090187/workflow#L33-L42 for an alternative.